sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/AutoConfiguredOpenTelemetrySdkBuilder.java

```diff
@@ -43,14 +43,14 @@
 
   private Supplier<Map<String, String>> propertiesSupplier = Collections::emptyMap;
 
+  private ClassLoader serviceClassLoader =
```

Comment:
This isn't the context classloader anymore - most of what I know of SPI is from the Armeria codebase where this is always the default we use, I have a feeling it has less gotchas than context, though both will naturally have some gotchas so we allow customizing.